### ISSUE TITLE
Cython is missing executables

### ISSUE THREAD

**Issue by deliciouslytyped** [NixOS/nixpkgs#307596]
For example, these exist in the cython source tree but I cant find them anywhere on my machine or with nix-locate.
- https://github.com/cython/cython/blob/8d5eb48473db9fda485c215683a6f22d1bb91c08/bin/pcython
- https://github.com/cython/cython/blob/8d5eb48473db9fda485c215683a6f22d1bb91c08/bin/cythonrun

xref https://github.com/cython/cython/issues/6170

cython packaged at https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/python-modules/cython

**Comment by Artturin** [comment]
Alpine doesn't install those either so should be a upstream issue https://pkgs.alpinelinux.org/contents?file=&path=&name=cython&branch=edge&repo=main&arch=x86_64

**Comment by deliciouslytyped** [comment]
Yeah, after looking into this some more I forgot to update the issue. I'm not familiar with python packaging but it seems like whatever python binary packages are these days, the scripts and 'entry_points' directives in setup.py result in their implicit inclusion and installation:

https://github.com/cython/cython/blob/8d5eb48473db9fda485c215683a6f22d1bb91c08/setup.py#L67-L79

**Comment by Artturin** [comment]
Submit upstream and see what they say?

**Comment by Artturin** [comment]
> Submit upstream and see what they say?

Bump
